p-token/Cargo.toml

```diff
@@ -15,8 +15,8 @@ crate-type = ["cdylib"]
 logging = []
 
 [dependencies]
-pinocchio = { version = "0.7", git = "https://github.com/febo/pinocchio.git", branch = "febo/close-unstable" }
-pinocchio-log = { version = "0.3", git = "https://github.com/febo/pinocchio.git", branch = "febo/close-unstable" }
+pinocchio = "0.8"
+pinocchio-log = { version = "0.4", default-features = false }
```

Comment:
Disabling default features to avoid bringing the log macro dependencies – the macro is not used.